cmd/root.go

```diff
@@ -127,7 +126,6 @@ func setFlagErrorFunc(cmd *cobra.Command, err error) error {
 	if e := cmd.Help(); e != nil {
 		return e
 	}
-	fmt.Println("")
```

Comment:
These were here for a reason, to separate the help text from the error message as a line-break.

Reply:
This is easily mistakable for a commit error, especially as time passes and our memory of this PR fades. A comment would communicate its intentional.